test/index.js

```diff
@@ -7,9 +7,9 @@ import { normalize, Schema, arrayOf, unionOf } from 'normalizr';
 
 describe("denormalize", () => {
 
-  it("should return undefined when denormalizing an undefined entity", () => {
-    expect(denormalize(undefined)).to.be.undefined;
-  });
+  // it("should return undefined when denormalizing an undefined entity", () => {
+  //   expect(denormalize(undefined)).to.be.undefined;
+  // });
```

Comment:
Hi @jcarbo thanks, looks like a great improvement. I'll review the PR tomorrow (it's nighttime here).
Is there a reason why this test has been commented out?